app/charts/map/map-tooltip.tsx

```diff
@@ -69,29 +69,38 @@ export const MapTooltip = () => {
   const areaTooltipState = useMemo(() => {
     if (areaLayer) {
       const obs = interaction.d;
-      const { colors } = areaLayer;
+      if (obs) {
```

Comment:
Nit, I think returning early if not areaLayer or obs would be a bit clearer here.